python/lsst/afw/math/backgroundList.py

```diff
@@ -152,6 +152,7 @@ def readFits(fileName, hdu=0, flags=0):
 
         self = BackgroundList()
 
+        hdu -= 1                        # we want to start at 0 (post RFC-304), but are about to increment
         while True:
```

Comment:
Two things here: 

1. Please make the comment 2 spaces from the code.
2. Ugh, really? This `while`-loop looks all kinds of wonky to me. I guess redoing it is out of scope, though.